cclient/serialization/json/json_serializer.c

```diff
@@ -455,3 +508,262 @@ size_t json_get_transactions_to_approve_deserialize_response_get_size(
     const serializer_t* const s, const char* const toDeserialize) {
   return 0;
 }
+
+// TODO
+// addNeighbors
+size_t json_add_neighbors_serialize_request_get_size(
+    const serializer_t* const s) {
+  return 0;
+}
+void json_add_neighbors_serialize_request(const serializer_t* const s,
+                                          char* out) {}
+
+retcode_t json_add_neighbors_deserialize_response(const serializer_t* const s,
+                                                  const char* const obj,
+                                                  int* out) {
+  cJSON* json_obj = cJSON_Parse(obj);
+  cJSON* json_value = NULL;
+
+  if (json_obj == NULL) {
+    cJSON_Delete(json_obj);
+    return RC_CCLIENT_JSON_PARSE;
+  }
+
+  json_value = cJSON_GetObjectItemCaseSensitive(json_obj, "error");
+  if (cJSON_IsString(json_value) && (json_value->string != NULL)) {
+    cJSON_Delete(json_obj);
+    return RC_CCLIENT_JSON_PARSE;
+  }
+
+  json_value = cJSON_GetObjectItemCaseSensitive(json_obj, "addedNeighbors");
+  if (!cJSON_IsNumber(json_value)) {
+    cJSON_Delete(json_obj);
+    return RC_CCLIENT_JSON_PARSE;
+  }
+
+  *out = json_value->valueint;
+  return RC_OK;
+}
+
+size_t json_add_neighbors_deserialize_response_get_size(
+    const serializer_t* const s, const char* const toDeserialize) {
+  return 0;
```

Comment:
not implemented

Reply:
as we discussed, may remove this function